samples/dnn/dnn_model_runner/dnn_conversion/paddlepaddle/README.md

```diff
@@ -0,0 +1,80 @@
+# Run PaddlePaddle model by OpenCV
+
+This tutorial gives 2 demos show how to run PaddlePaddle model by opencv
+
+## Environment Setup
+
+```shell
+pip install paddlepaddle-gpu
+pip install paddlehub
+pip install paddle2onnx
+```
+
+## 1. Run PaddlePaddle ResNet50 by OpenCV
+
+### Run PaddlePaddle model demo
+
+Run the example code as below,
+
+```shell
+python paddle_resnet50.py
+```
+
+there are 3 part of this execution
+
+* 1. Export PaddlePaddle ResNet50 model to onnx format;
+* 2. Use `cv2.dnn.readNetFromONNX` load model file;
+* 3. Preprocess image file and do inference.
+
+## 2. Run PaddleSeg Portrait Segmentation by OpenCV
+
+### Convert to ONNX Model
+
+#### Step1. get Paddle Inference model
+
+Thanks for [PaddleSeg](https://github.com/PaddlePaddle/PaddleSeg), this Paddle Inference model convert from HRNet w18 small v1. For more details, please refer to [HumanSeg](https://github.com/PaddlePaddle/PaddleSeg/blob/release/2.1/contrib/HumanSeg/README.md).
+
+```shell
+wget https://x2paddle.bj.bcebos.com/inference/models/humanseg_hrnet18_small_v1.zip
+unzip humanseg_hrnet18_small_v1.zip
+```
+
+Notes:
+
+* The exported Paddle Inference model has to be fixed input shape while dynamic shape is not supported by opencv now.
```

Comment:
* The exported model must have a fixed input shape, as dynamic is not supported at this moment.

Reply:
Done.